packages/admin-ui/ui/src/hooks/use-build-timeline.tsx

```diff
@@ -175,6 +177,12 @@ export const useBuildTimeline = (orderId: string) => {
 
   const { notifications } = useAdminNotifications({ resource_id: orderId })
 
+  const { stock_locations } = useAdminStockLocations()
+
+  const getLocationNameById = (locationId: string | null) =>
```

Comment:
nit(non-blocking): since this is declared multiple times we could move this to the domain folder, e.g. `domain/inventory/location/utils`

Reply:
Ah great shout, I did think I should put it somewhere better but wasn't sure where 😅

Reply:
Made it part of its own `useStockLocations` hook, that currently only queries locations and provides those and this function, but could be a decent util hook to extend as needed. Wdyt?

Reply:
Sounds good 👍